src/graphnet/data/extractors/i3featureextractor.py

```diff
@@ -122,14 +149,24 @@ def _get_relative_dom_efficiency(self, frame, om_key):
 
 
 class I3FeatureExtractorIceCubeDeepCore(I3FeatureExtractorIceCube86):
-    """..."""
+    """Class for extracting reconstructed features for IceCube-DeepCore."""
```

Comment:
I'm not sure an IceCuber will recognize what is meant with "reconstructed features" here. I think "Class for extracting unfolded waveform pulses and associated OM information, such as position, orientation, etc." is more descriptive. 

I don't think you necessarily have to change all the doc strings accordingly in this PR. Perhaps it's best if we discuss language first and iterate on doc string wording in a separate PR in the future.

Reply:
I think this is a good point, and this PR might be a good time to review our language as it will update all docstrings in the codebase. I'll try to keep this in mind going forward!